pom.xml

```diff
@@ -86,6 +128,97 @@
 				</pluginRepository>
 			</pluginRepositories>
 		</profile>
+		<!-- Formatting setup, inherited by all projects using this parent -->
+		<profile>
+			<id>format</id>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+				<property>
+					<name>!no-format</name>
+				</property>
+				<!-- This condition makes sure the ide-config can have parent set and the formatting is skipped -->
+				<!-- for all modules that don't contain actual code (such as spec text or this parent) -->
+				<file>
+					<exists>src/main/java</exists>
+				</file>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>net.revelc.code.formatter</groupId>
+						<artifactId>formatter-maven-plugin</artifactId>
+						<executions>
+							<execution>
+								<phase>process-sources</phase>
+								<goals>
+									<goal>format</goal>
+								</goals>
+							</execution>
+						</executions>
+					</plugin>
+					<plugin>
+						<groupId>net.revelc.code</groupId>
+						<artifactId>impsort-maven-plugin</artifactId>
+						<executions>
+							<execution>
+								<id>sort-imports</id>
+								<goals>
+									<goal>sort</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<removeUnused>true</removeUnused>
+						</configuration>
```

Comment:
```suggestion
```

This is inherited from `<pluginManagement>`, no?

Reply:
Good catch, that's an oversight and same was in validation profile.
Should be fixed now.